tensorflow_datasets/image/flic.py

```diff
@@ -0,0 +1,101 @@
+"""Frames Labeled In Cinema (FLIC)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import scipy.io
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_CITATION = """@inproceedings{modec13,
+    title={MODEC: Multimodal Decomposable Models for Human Pose Estimation},
+    author={Sapp, Benjamin and Taskar, Ben},
+    booktitle={In Proc. CVPR},
+    year={2013},
+  }
+"""
+
+_DESCRIPTION = """
+From the paper: We collected a 5003 image dataset automatically from popular
+Hollywood movies. The images were obtained by running a state-of-the-art person
+detector on every tenth frame of 30 movies. People detected with high confidence
+(roughly 20K candidates) were then sent to the crowdsourcing marketplace Amazon
+Mechanical Turk to obtain groundtruthlabeling. Each image was annotated by five
+Turkers for $0.01 each to label 10 upperbody joints. The median-of-five labeling
+was taken in each image to be robust to outlier annotation. Finally, images were
+rejected manually by us if the person was occluded or severely non-frontal. We
+set aside 20% (1016 images) of the data for testing. 
+"""
+
+_URL = "https://drive.google.com/uc?id=0B4K3PZp8xXDJN0Fpb0piVjQ3Y3M&export=download"
+
+class Flic(tfds.core.GeneratorBasedBuilder):
+  """Frames Labeled In Cinema (FLIC)"""
+
+  VERSION = tfds.core.Version('1.0.0')
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=_DESCRIPTION,
+        features=tfds.features.FeaturesDict({
+            "image": tfds.features.Image(),
```

Comment:
As far as I remember the shape of all the images are more or less same. So, you can put that shape here as:
`tfds.features.Image(shape=(height, width, 3))`